airflow/providers/google/provider.yaml

```diff
@@ -542,36 +542,36 @@ hooks:
 transfers:
   - source-integration-name: Presto
     target-integration-name: Google Cloud Storage (GCS)
-    how-to-guide: /docs/howto/operator/google/transfer/presto_to_gcs.rst
+    how-to-guide: /docs-providers/apache-airflow-providers-google/operators/transfer/presto_to_gcs.rst
     python-module: airflow.providers.google.cloud.transfers.presto_to_gcs
   - source-integration-name: SQL
     target-integration-name: Google Cloud Storage (GCS)
     python-module: airflow.providers.google.cloud.transfers.sql_to_gcs
   - source-integration-name: Google Cloud Storage (GCS)
     target-integration-name: Google Drive
-    how-to-guide: /docs/howto/operator/google/transfer/gcs_to_gdrive.rst
+    how-to-guide: /docs-providers/apache-airflow-providers-google/operators/transfer/gcs_to_gdrive.rst
     python-module: airflow.providers.google.suite.transfers.gcs_to_gdrive
   - source-integration-name: Microsoft SQL Server (MSSQL)
     target-integration-name: Google Cloud Storage (GCS)
     python-module: airflow.providers.google.cloud.transfers.mssql_to_gcs
   - source-integration-name: Microsoft Azure FileShare
     target-integration-name: Google Cloud Storage (GCS)
-    how-to-guide: /docs/howto/operator/google/transfer/azure_fileshare_to_gcs.rst
+    how-to-guide: /docs-providers/apache-airflow-providers-google/operators/transfer/azure_fileshare_to_gcs.rst
     python-module: airflow.providers.google.cloud.transfers.azure_fileshare_to_gcs
   - source-integration-name: Apache Cassandra
     target-integration-name: Google Cloud Storage (GCS)
     python-module: airflow.providers.google.cloud.transfers.cassandra_to_gcs
   - source-integration-name: Google Spreadsheet
     target-integration-name: Google Cloud Storage (GCS)
-    how-to-guide: /docs/howto/operator/google/transfer/sheets_to_gcs.rst
+    how-to-guide: /docs-providers/apache-airflow-providers-google/operators/transfer/sheets_to_gcs.rst
     python-module: airflow.providers.google.cloud.transfers.sheets_to_gcs
   - source-integration-name: Amazon Simple Storage Service (S3)
     target-integration-name: Google Cloud Storage (GCS)
-    how-to-guide: /docs/howto/operator/google/transfer/s3_to_gcs.rst
+    how-to-guide: /docs-providers/apache-airflow-providers-google/operators/transfer/s3_to_gcs.rst
```

Comment:
Do we need the whole path here?  I think just sheets_to_gcs should be needed and all the rest should be a convention

Reply:
Now we definitely need a full path because some documentation is in the core and some in providers. In the long term, I also think the full path seems to be helpful to me, as new contributors will find it easier to find this file.